mllib/src/main/scala/org/apache/spark/mllib/clustering/GaussianMixtureModel.scala

```diff
@@ -83,5 +95,87 @@ class GaussianMixtureModel(
       p(i) /= pSum
     }
     p
-  }  
+  }
+}
+
+@Experimental
+object GaussianMixtureModel extends Loader[GaussianMixtureModel] {
+
+  object SaveLoadV1_0 {
+
+    case class Data(weights: Array[Double], mus: Array[Vector], sigmas: Array[Array[Double]])
```

Comment:
Shall we flatten them? So if one inspect the Parquet file, they can see each center as a record. Btw, I'm not sure whether we should implement `MatrixUDT` first or use dense arrays here.